src/minimal.rs

```diff
@@ -37,8 +38,11 @@ pub struct MinimalSecureLayer {
     pub(crate) encrypt_algo_with_secret: Option<EncryptAlgoWithSecretKey>,
     ephemeral_kp: Option<EphemeralKeyPair>,
     pub(crate) ephemeral_pubkey: EphemeralPublicKey,
+    nonce_list: BTreeSet<u64>,
```

Comment:
Not explicit enough, we do not understand the role of this list. To be renamed to `orphan_nonces_list`.
Also adds a comment that explains the role of this field.